### DISCUSSION TITLE
Realtime Messaging to Change Input Filter Overlays?

### DISCUSSION THREAD

**Discussion by mikekoetter** [CESNET/UltraGrid#377] [Ideas]
Has there been any thought given to a way of altering the Input Filter overlays like Text and Logo with some sort of realtime messaging?  If say you had a metadata server that could signal when a clip changes to update the Text overlay to a new Sequence and Clip name.  Or if you had something like an ATEM or Streamdeck that you could use to change a lower thirds logo in realtime.

**Comment by alatteri** [comment]
Hi Mike,

We do lots of VFX reviews utilizing Flame and Resolve.  Both software have the ability to turn on Metadata overlays which contain the information you suggest and update in real time.   It seems more appropriate that the playback application does this, than the encoding application.

**Reply by mikekoetter** [reply]
Well for your example you would need to have the overlays always turned on in your Color Suite or review room, which for some colorist would be a non-starter, where as the receiving end production team may prefer it.

**Reply by alatteri** [reply]
valid point.  I don't know if you ever worked with Lustre, but it has a web server built in which displays that information in realtime and does not require SDI overlay.

**Comment by TheSashmo** [comment]
I could say that would apply on both sides, not just one.  Depending on your model.
